src/main/kotlin/io/gitlab/arturbosch/detekt/config/DetektConfigStorage.kt

```diff
@@ -13,7 +13,7 @@ import com.intellij.util.xmlb.annotations.Tag
  */
 @State(
     name = "DetektProjectConfiguration",
-    storages = [(Storage(StoragePathMacros.WORKSPACE_FILE))]
+    storages = [Storage("detekt.kml")]
```

Comment:
kml is the right format?
And it is put by convention into the .idea folder?

Reply:
oops, I meant `.xml` obviously! Thanks for catching it.

Yes, path is relative to workspace by default, the file is created next to workspace.xml.